packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -1468,6 +1468,26 @@
     "dynamic": false,
     "properties": {}
   },
+  "cases-oracle": {
```

Comment:
This is a new saved object type that we will use for the case action. This PR will be merged on a feature branch and the definition may change as we develop.

Reply:
Aren't we already sure the `grouping_definition` should be included?

Reply:
We will not index the [`grouping_definition`](https://github.com/elastic/kibana/pull/168370/files/c210a0c9cefc167ac1c1339757ec8494729abb4e#diff-b37650ad2dc0ea1bab96cf67e8cf09fd50638f9c7d39d51dc0a4b191df948f34R33) so it will not be included in the mapping.